public/css/indexcss.css

```diff
@@ -0,0 +1,199 @@
+*{
+	margin:0;
+	padding:0;
+}
+body{
```

Comment:
I don't think overriding body is a good idea, why not use the topmost div in the template as a selector